cellmlmanip/model.py

```diff
@@ -599,18 +599,19 @@ def _set_variable_type(variable, variable_type):
             logger.warning('Variable %s already has type=="%s". Skip setting "%s"',
                            variable.dummy, variable.type, variable_type)
 
-    def get_symbols(self, expr):
-        """Returns the symbols in an expression. Works differently to Sympy atoms() because a
+    def get_symbols(self, expressions):
+        """Returns the symbols in a collection of expressions. Works differently to Sympy atoms() because a
```

Comment:
Please rewrite this a bit clearer. I thought it was saying that Sympy's `atoms()` regarded a derivative as a single symbol, but it seems from the code that you mean the opposite.

Reply:
To be honest, don't mention Sympy at all (we're not documenting Sympy, plus what if they change their code?)
Just add a note saying that derivatives count as a single symbol.

Reply:
I just changed what was changed but I don't mind updating the doc string